attributes/_install.rb

```diff
@@ -13,16 +13,21 @@
     '/etc/splunk'
   end
 
-default['splunk']['package']['version'] = '9.0.2'
-default['splunk']['package']['build'] = '17e00c557dc1'
+default['splunk']['package']['version'] = '9.0.5'
+default['splunk']['package']['build'] = 'e9494146ae5c'
 default['splunk']['is_cloud'] = false
 default['splunk']['package']['base_url'] = 'https://download.splunk.com/products'
 default['splunk']['package']['platform'] = node['os']
 default['splunk']['package']['file_suffix'] =
   case node['platform_family']
   when 'rhel', 'fedora'
     if node['kernel']['machine'] == 'x86_64'
+      # linux rpms of splunk/UF before 9.0.5 are a differently named package.
+      if default['splunk']['package']['version'] >= '9.0.5'
```

Comment:
This doesn't work in all cases.  For an accurate comparison of version numbers you need to use Gem::Version.
https://code.dblock.org/2020/07/16/comparing-version-numbers-in-ruby.html
https://metaredux.com/posts/2018/10/28/a-better-way-to-compare-versions-in-ruby.html
https://github.com/cerner/cerner_splunk/blob/383a28b5526804a13cddf819b7ab7fd909abc492/recipes/_start.rb#L13-L16

Reply:
Added: https://github.com/cerner/cerner_splunk/pull/270/commits/cc3d743fccdba2445322debbd8159a1c71ee1051

rspec passed, s_standalone, and all f_* boxes passed.

Updated vagrant to 2.3.7 while working on ops_splunk_roles which caused issues with the windows box. Apparently its fixed in 2.3.8 previews, I downgrade to 2.3.6 in the meantime and all is well.